metricbeat/module/kvm/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: kvm
+  metricsets: ["dommemstat"]
+  enabled: false
+  period: 10s
+  hosts: ["localhost"]
+
+  # Timeout to connect to Libvirt server
+  #timeout: 1s
```

Comment:
@ruflin we will have to add a reference config and default metricsets, as this module is experimental I think this can be left for a future PR